frontend/app/src/store/assets/asset-cache.ts

```diff
@@ -66,6 +66,9 @@ export const useAssetCacheStore = defineStore('assets/cache', () => {
         logger.debug(`unknown key: ${key}`);
         unknown.set(key, Date.now() + CACHE_EXPIRY);
       }
+    } catch (e) {
+      logger.debug(`unknown key: ${key}`);
+      unknown.set(key, Date.now() + CACHE_EXPIRY);
```

Comment:
In which case do we have this happening? backend error?

Reply:
Do you have an example response? or something

Reply:
Got this 
```
data: {
  message: "One or more of the given identifiers could not be found in the database"
  result: null
}
```